### ISSUE TITLE
Allow backwards rotation

### ISSUE THREAD

**Issue by kkirby** [Bargest/LogicExtensions#6]
https://github.com/Bargest/LogicExtensions/blob/22b30e03f54e199676bd84098b2804f5881d6efd/logic_mod/BlockScripts/CogScript.cs#L41

Any idea how this could be modified to allow backwards rotation? I'm unfamiliar with Besiege's API, but if all you're doing to go forward is setting the speed slider to a positive value, then I'd think you could go backwards by using a negative number. So, maybe something like this:

```C#
cogMotor.SpeedSlider.Value = Mathf.Lerp(-savedSpeed, savedSpeed, Math.Abs(forward - backward));
```

**Comment by Bargest** [comment]
This would be wrong, since if `forward = 0.5` and `backward = 0` we'll get `Lerp(-speed, +speed, 0.5)` which is middle of `[-speed, +speed] == 0`. I think it should be something like
```csharp
var diff = forward - backward;
var direction = diff >= 0 ? 1 : -1;
cogMotor.SpeedSlider.Value = Mathf.Lerp(0, savedSpeed * direction, Math.Abs(diff));
```
Thanks for reporting this bug, I'll publish update in several days.

BTW modifying speed slider at runtime is not how Besiege's API is supposed to be used, it is kind of dirty hack. API does not provide enough functionality for controlling blocks. Luckily game does not crash on negative speed slider values.

**Comment by Bargest** [comment]
Fixed in ed2376700f25242bb28714d4241f3a489cb17f50
